corehq/apps/export/static/export/js/download_export.ng.js

```diff
@@ -0,0 +1,320 @@
+(function (angular, undefined) {
+    'use strict';
+    // module: hq.download_export
+
+    /* This is the helper module for the Download Exports Page. */
+
+    var download_export = angular.module('hq.download_export', [
+        'ngResource',
+        'ngRoute',
+        'ng.django.rmi',
+        'ngMessages'
+    ]);
+
+    var $element = {
+        progress: function () {
+            return $('#download-progress-bar');
+        },
+        group: function () {
+            return $('#id_group');
+        },
+        user_type: function () {
+            return $('#id_user_types');
+        }
+    };
+
+    download_export.constant('exportList', []);
+    download_export.constant('maxColumnSize', 2000);
+    download_export.constant('defaultDateRange', null);
+    download_export.constant('checkForMultimedia', false);
+
+    var exportsControllers = {};
+    exportsControllers.DownloadExportFormController = function (
+        $scope, djangoRMI, exportList, maxColumnSize, exportDownloadService,
+        defaultDateRange, checkForMultimedia
+    ) {
+        var self = {};
```

Comment:
isn't `self` always supposed to a reference to `this`? just curious why you decided to change